code/game/objects/items.dm

```diff
@@ -1141,7 +1155,7 @@ modules/mob/living/carbon/human/life.dm if you die, you will be zoomed out.
 ///Generates worn icon for sprites on-mob.
 /obj/item/proc/make_worn_icon(species_type, slot_name, inhands, default_icon, default_layer)
 	//Get the required information about the base icon
-	var/icon/icon2use = get_worn_icon_file(species_type = species_type, slot_name = slot_name, default_icon = default_icon, inhands = inhands)
+	var/icon2use = get_worn_icon_file(species_type = species_type, slot_name = slot_name, default_icon = default_icon, inhands = inhands)
```

Comment:
> Whats the point of some of these images -> MA changes anyway

Images generate appearances when they're modified so everytime you make changes to images they have to do that, while with MA's they don't and just generate it when applied as a overlay or something(which then a immutable appearance is made), supposedly that gives performance gains but i dind't perform a test, however TGStation has switched to use MA's on this part of code since 2017 so in my view it seems worth